toolbox/metrics/cmd/service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    istio: metrics-fetcher
+  name: metrics-fetcher
+  namespace: metrics
+spec:
+  ports:
+  - port: 9103
```

Comment:
This is not an ask but I see multiple hardcoded ports 9103 in the yamls and in go files and it would be nice to have a single copy of config.

Reply:
in go this is a flag. I am not I understand what you want.